src/main/resources/handlebars/soknad.hbs

```diff
@@ -306,15 +306,13 @@
     <section id="utenlandskNæring">
         {{#if hjelper.utenlandskNæringSatt}}
             <h2>Utenlandsk næring</h2>
-            <h3>Har du jobbet som selvstendig næringsdrivende i et annet EØS-land i løpet av de 3 siste årene før perioden du søker om?</h3>
             {{#if utenlandskNæring}}
-                <p>Ja</p>
                 {{#each utenlandskNæring as |næring|~}}
                     <ul>
                         <li>
                             <p><b>{{næring.navnPåVirksomheten}} ({{næring.fraOgMed}} - {{næring.tilOgMed}})</b></p>
                             <p>{{næring.land.landnavn}} {{næring.land.landkode}}</p>
-                            <p>Identifikasjonsnummer: {{næring.identifikasjonsnummer}}</p>
+                            <p>Organisasjonsnummer: {{næring.identifikasjonsnummer}}</p>
```

Comment:
Er dette riktig? 
Organisasjonsnummer: næring.`identifikasjonsnummer `

Reply:
Jeg kan refaktorere hele feltnavnet. Litt dårlig kommunikasjon over hva feltet egentlig var

Reply:
Er ikke krise, men ble litt forvirret over de to navnene :D

Reply:
Hehe ja enig at det er forvirrende, fikser ✅